src/app/common/subscriptions/SubscriptionDetails.vue

```diff
@@ -29,25 +29,30 @@
     </div>
 
     <div v-if="detailsIterator">
-      <ul class="overview-stat-grid">
-        <li
+      <ul
+        class="columns"
+        style="--columns: 4;"
```

Comment:
There isn’t a clear alternative to using `style` that would allow the same degree of freedom. One could prepare dedicated rule sets like `.columns--4 { --columns: 4; }` and use them instead, but that could only ever be finite.

Seeing as we currently don’t set a CSP with style-src, I’d rather keep the flexibility now and change it if such a rule is introduced (in which case the changes would be trivial).

Reply:
Cool yeah no problem to leave for the minute, its not like we do CSP at present.

> There isn’t a clear alternative to using style that would allow the same degree of freedom. 

An alternative could be something like (the semantic class naming/selector could be anything or course):

```css
.subscription-details .information ul {
  --columns: 4
}

```

Not sure what degree of freedom this doesn't give us compared to using `style`, but I could be missing something.

Just to be clear, no action needed from me here unless you specifically wanted to change it yourself